cli/src/installers/dependencyVersionMap.ts

```diff
@@ -11,6 +11,13 @@ export const dependencyVersionMap = {
   prisma: "^4.14.0",
   "@prisma/client": "^4.14.0",
 
+  // Drizzle
+  "drizzle-orm": "^0.26.3",
+  "drizzle-kit": "^0.18.1",
+  dotenv: "^16.1.4",
+  // REVIEW: Is it fine including pscale by default?
```

Comment:
not sure how i feel about this considering the opinion we've had in the past that the "core stack" should always be free. 

is the reasoning here that this is our best way to be edge ready?

Reply:
idk we could take it one step further and prompt for what driver they want - but that's though when it comes to the nextauth adapter as it will differ a bit.

until nextauth ships the adapter and handles all of this for you, maybe we can just go for pscale?

the biggest selling point here is the edge compat and selecting the normal mysql driver wouldn't make sense for that.

idk what's best - hence i left the comment pointing it out - but definetely something to think about